application/helpers/expressions/em_manager_helper.php

```diff
@@ -8518,11 +8520,6 @@ static function ProcessCurrentResponses()
                         {
                             $value = (isset($_POST[$sq]) ? $_POST[$sq] : '');
                         }
-                        if ($radixchange && isset($LEM->knownVars[$sq]['onlynum']) && $LEM->knownVars[$sq]['onlynum']=='1')
-                        {
-                            // convert from comma back to decimal
-                            $value = implode('.',explode(',',$value));
-                        }
```

Comment:
You remove it here ;) and remind "text" with ad attribute "only num". And Array number ;)